migrationDockerfile

```diff
@@ -0,0 +1,9 @@
+FROM usgswma/python:3.8
+
+COPY requirements.txt .
+COPY requirements-dev.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt \
+    && pip install --no-cache-dir -r requirements-dev.txt \
```

Comment:
I don't think you should use requirements-dev.txt? So is there anyway to use the other Dockerfile rather than create two?

Reply:
I'm can requirements-prod instead. I've tried to use the existing Dockerfile, but the stage is neither happy with the `CMD` nor have attempts to override it have been effective (they've just caused other errors).

I did try to use a Docker image instead of file, but ran into permissions issues that I can't resolve without altering the base image. Adding an additional Dockerfile, seemed like the lesser of the various evils.